docs/pages/database-access/guides/sql-server-ad.mdx

```diff
@@ -369,7 +369,8 @@ $ tsh db connect --db-user=teleport sqlserver
   you're running `tsh db connect` from.  
   
   `mssql-cli` is not required for SQL Server connections.  Use `tsh proxy db --db-user=teleport --tunnel sqlserver`
-   to connect from other DB Clients such as Microsoft SQL Server Management Studio.
+   to connect from other DB Clients such as Microsoft SQL Server Management Studio. See the [Database Access GUI Clients
```

Comment:
I think we should move this new paragraph and the link to a "Next steps" section since following the link is not required to complete the guide.

Reply:
@stevenGravy What do you think of the comment above?

Reply:
Given that very few SQL Server users have `mssql-cli` I think we should keep it.  It's also very difficult to install at times since it depends on python and is not a standalone binary. This has been a frequent trip up when attempting to use SQL Server access with Teleport.

Reply:
As it stands now, this Admonition is pretty confusing, since it tells the reader that `mssqli-cli` should be in PATH, then in the next paragraph, that it's not required. I think we can be a lot more explicit here.

Can we say something like:

````markdown
When you run the `tsh db connect` command above, 
`tsh` attempts to execute the `mssql-cli` command-line 
client, which must be available in PATH. If you do not 
have `mssql-cli` available on your system, you can run 
the following command to start a local proxy server that 
you can connect to with your SQL Server client:

```code
$ tsh proxy db --db-user=teleport --tunnel sqlserver
```

Read the [Database Access GUI Clients](../../connect-your-client/gui-clients.mdx#sql-server-azure-data-studio) 
guide for how to connect your GUI client to the local proxy.
````

This also indicates to the reader what they should expect to get out of the GUI Clients guide, rather than saying "more information".

Reply:
Please see update @ptgott